### PR TITLE
Percent refactor

### PR DESCRIPTION
**Pull request by Paul-Knapp**
percents have all been refactored to match new standard for testing purposes with the appropriate testing changes

### CODE REVIEW

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -39,11 +39,81 @@ def self.from_csv(source)
...
+      team_games[team_id] += 1
+      end
+  
+      team_averages = team_goals.map do |team_id, total_goals|
+        games_played = team_games[team_id]
+        average_goals = total_goals.to_f / games_played
+        [team_id, average_goals]
+      end
+  
+      best_team_id = team_averages.min_by { |_,avg| avg }.first
```
Comment (wally-yawn) [inline lib/stat_tracker.rb:113]: This variable name is probably meant to be 'worst' not 'best'

File: lib/stat_tracker.rb [hunk 2]
```diff
@@ -39,11 +39,81 @@ def self.from_csv(source)
...
+      best_team = @all_teams.find {|team| team.team_id == best_team_id}
```
Comment (wally-yawn) [inline lib/stat_tracker.rb:114]: This variable name is probably meant to be 'worst' not 'best'

File: lib/stat_tracker.rb [hunk 3]
```diff
@@ -39,11 +39,81 @@ def self.from_csv(source)
...
+  
+      best_team.teamName
```
Comment (wally-yawn) [inline lib/stat_tracker.rb:116]: This variable name is probably meant to be 'worst' not 'best'

**Review by wally-yawn — CHANGES_REQUESTED** [review]
I've left comments about variable names that should be resolved before merging. Additionally lines 93-111 in stat_tracker are duplicates of each other, this could probably be refactored into a helper method so best/worst_offenses only have to call max_by or min_by. it's fine for now but maybe create a trello card to refactor that if we have time?

**Comment by Paul-Knapp** [comment]
variable names have been changed and will refactor with a helper on a different branch

**Pull request closed without merging**